src/schemas/v2/combined.json

```diff
@@ -19633,41 +19977,67 @@
                 }
               }
             },
-            "centralHeating": {
+            "heating": {
+              "ntsRef": "B3.4",
               "baspiRef": "A7.4",
               "ta6Ref": "12.3",
               "rdsRef": "Buildings|BuildingSections/isUnheated=no|yes|true|false",
               "piqRef": "A7.3",
-              "title": "Central Heating",
+              "title": "Heating",
               "type": "object",
-              "baspiRequired": ["hasCentralHeating"],
+              "baspiRequired": ["heatingType"],
+              "ta6Required": ["heatingType"],
+              "ntsRequired": ["heatingType", "otherHeatingFeatures"],
               "properties": {
-                "hasCentralHeating": {
+                "heatingType": {
+                  "ntsRef": "B3.4.1",
                   "baspiRef": "A7.4.0",
                   "ta6Ref": "12.3.1",
                   "piqRef": "A7.3.1",
-                  "title": "Is there full/partial central heating in your property?",
+                  "title": "What type of heating system does the property have?",
                   "type": "string",
-                  "enum": ["Yes", "No"]
+                  "enum": ["Central heating", "Communal heating system", "None"]
```

Comment:
Is other a valid option here?

Reply:
Should provide room heaters as an option (no need to capture fuel type) , also include "Other" with free text details, remove "none".